src/screens/SUE/SueMapViewScreen.tsx

```diff
@@ -0,0 +1,131 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { useContext, useEffect, useState } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { SettingsContext } from '../../SettingsProvider';
+import { Map } from '../../components';
+import { colors, normalize } from '../../config';
+
+export const SueMapViewScreen = ({
+  navigation,
```

Comment:
not used

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1140/commits/e01b9da4ca5a57b77cbd45cf023113332406e9a6